### DISCUSSION TITLE
[vcpkg] Cannot install any lib with vcpkg on Visual Studio Code.

### DISCUSSION THREAD

**Discussion by AntoineYK** [microsoft/vcpkg#33653] [Q&A] [answered]
![image](https://github.com/microsoft/vcpkg/assets/107028667/766c5a15-c88d-4044-a2f5-50c172dda932)
Here is what happens when I try to install a lib. 
I am only using Visual Studio Code on Windows 10 64x

Can you help me to try to solve this issue please ? 

Thank you !

**Comment by autoantwort** [comment, marked as answer]
_Visual Studio Code_ is not _Visual Studio_, you have to install "Visual Studio"
